src/table/components/TableWrapper.jsx

```diff
@@ -50,14 +50,19 @@ export default function TableWrapper(props) {
     keyboard,
     rect,
     announcer, // this is only for testing purposes
+    theme,
   } = props;
   const { size, rows, columns } = tableData;
   const { page, rowsPerPage, rowsPerPageOptions } = pageInfo;
   const [focusedCellCoord, setFocusedCellCoord] = useState([0, 0]);
   const shouldRefocus = useRef(false);
   const tableContainerRef = useRef();
   const tableWrapperRef = useRef();
-  const classes = useStyles();
+
+  const backgroundColorFromTheme = theme.getStyle('object', 'straightTable', 'backgroundColor');
```

Comment:
Unless `'straightTable'` is used for view data we might want to consider having a separate property for this table. Name TBD in that case I suppose